common/wire_error.c

```diff
@@ -2,6 +2,7 @@
 #include <ccan/tal/str/str.h>
 #include <common/type_to_string.h>
 #include <common/wire_error.h>
+#include <netinet/in.h>
```

Comment:
I can't see why this one is needed?

Reply:
I'm going to apply it without this change.

I tried to build on FreeBSD but it was beyond my skill and doc/INSTALL doesn't say how to do it.

Reply:
Huh, I'll try building it without this change and see what happens.